extension-list

```diff
@@ -95,6 +95,7 @@ $IP/extensions/MassMessage/extension.json
 $IP/extensions/Math/extension.json
 $IP/extensions/MediaWikiChat/extension.json
 $IP/extensions/MirahezeMagic/extension.json
+$IP/extensions/MobileDetect/MobileDetect.php
```

Comment:
Why not extension.json?

Reply:
The extension has not been updated to support extension.json yet.